tasks/modify_settings.yml

```diff
@@ -0,0 +1,33 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Configure setting with value
+  when: item.previous | d() != 'replaced'
+  vars:
+    __bootloader_with_value: >-
+      {{ item.name is defined and item.value is defined }}
+    __bootloader_setting: >-
+      {{ __bootloader_with_value |
+      ternary(item.name | string + '=' + item.value | d() | string,
```

Comment:
ok - I guess due to the order of evaluation, you need the `item.value | d()` - I think I had a similar issue with the kdump role